source/PlayerInfo.cpp

```diff
@@ -3672,6 +3672,14 @@ void PlayerInfo::RegisterDerivedConditions()
 	pluginProvider.SetHasFunction(pluginFun);
 	pluginProvider.SetGetFunction(pluginFun);
 
+	auto &&destroyedPersonProvider = conditions.GetProviderPrefixed("destroyed person: ");
+	auto destroyedPersonFun = [](const string &name) -> bool
+	{
+		return GameData::Persons().Has(name.substr(strlen("destroyed person: ")));
```

Comment:
Doesn't this return false if the person is dead? I think it should return true with this naming.